logs_sdk/Gemfile

```diff
@@ -11,6 +11,8 @@ gemspec
 eval_gemfile '../contrib/Gemfile.shared'
 
 group :test, :development do
+  gem 'mutex_m' if RUBY_VERSION >= '3.4'
```

Comment:
I take it this is required for testing rails logging bits?

Reply:
Good question. This isn't needed anymore. Minitest was missing a mutex_m requirement on the version in my Gemfile.lock. However, that version was outdated. Minitest 5.21+ removed the dependency. I'll remove this.
```suggestion
```